templates/cases/case_read.haml

```diff
@@ -92,8 +95,11 @@
                 opened this case and assigned it to
                 %a{ ng-href:"/partner/read/[[ event.item.assignee.id ]]/" }
                   [[ event.item.assignee.name ]]
-                %a{ ng-href:"/user/read/[[ event.item.user_assignee.id ]]/", ng-if:"event.item.user_assignee" }
-                  ([[ event.item.user_assignee.name ]])
+                %span{ ng-if:"event.item.user_assignee" }
```

Comment:
So that the parentheses around the username doesn't become part of the user link